clang/lib/DPCT/CallExprRewriter.h

```diff
@@ -978,7 +1001,7 @@ template <class SubExprT> class ZeroInitializerPrinter {
 };
 
 template <class FirstPrinter, class... RestPrinter>
-class MultiStmtsPrinter : MultiStmtsPrinter<RestPrinter...> {
+class MultiStmtsPrinter : public MultiStmtsPrinter<RestPrinter...> {
```

Comment:
Why "public" is needed here?

Reply:
`MultiStmtsPrinter<X1, X2, X3>::print` calls  `MultiStmtsPrinter<X2, X3>::printStmt`, but `MultiStmtsPrinter<X2, X3>::printStmt` is private (because `MultiStmtsPrinter<X3>::printStmt` is protected and `MultiStmtsPrinter<X2, X3>` privately inherits `MultiStmtsPrinter<X3>`). Public or protected inheritance makes `MultiStmtsPrinter<X2, X3>::printStmt` protected, which allows the method to be called.